src/framework/data/createIntegrationEntity.ts

```diff
@@ -202,7 +202,8 @@ function whitelistedProviderData(
   const whitelistedProviderData: ProviderSourceData = {};
   const schemaProperties = schemaWhitelistedPropertyNames(_class);
   for (const e of Object.entries(source)) {
-    if (schemaProperties.includes(e[0])) {
+    const hasValue = e[1] !== null && e[1] !== undefined;
```

Comment:
Slightly nitpicky, but I would recommend destructuring to make this more readable:

For example:

```typescript
const [key,value] of Object.entries(source)
```

Reply:
Simplification:

```typescript
const hasValue = value != null;
```

`!= null` is the one exception that you can make for using that operator because it will check for `null` and `undefined`.

Reply:
Good point, that does look better.